src/components/AppHeader.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class='app-header'>
+    <div class='ui secondary pointing menu'>
+      <router-link to='/' class='item'>Feed</router-link>
```

Comment:
`Feed` is a chronological feed of all things. `Following` is a list of people you are following, sorted by latest post by default. This will probably be sortable/filterable to quickly find someone.